embedded/embedded-api/src/main/java/org/mule/runtime/module/embedded/internal/MavenContainerClassLoaderFactory.java

```diff
@@ -8,6 +8,7 @@
 package org.mule.runtime.module.embedded.internal;
 
 import static java.util.stream.Collectors.toList;
+import static org.codehaus.plexus.util.FileUtils.toFile;
```

Comment:
Perhaps we could use https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#toFile(java.net.URL) instead of plexus utility code here.